### PR TITLE
Fix Docker publish

### PR DESCRIPTION
**Pull request by prryplatypus**

### CODE REVIEW

**Review by ahopkins — DISMISSED** [review]

File: .github/workflows/publish-release.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ name: Publish release
 on:
   release:
     types: [created]
-  workflow_dispatch:
```
Comment (ahopkins) [inline .github/workflows/publish-release.yml:6]: You don't think we want this on demand? How do you want to release the LTS images?

Comment (prryplatypus) [reply .github/workflows/publish-release.yml:6]: > You don't think we want this on demand?
The workflow currently will error when triggered manually. LTS patches can be created by creating the corresponding release and NOT marking it as `latest` in the release page.

> How do you want to release the LTS images?
For this version I'll upload them manually.

**Comment by prryplatypus** [comment]
Don't merge yet, there's another issue

**Comment by prryplatypus** [comment]
Images for v23.12.0 have been uploaded to Docker Hub. And fixes another couple of issues along the way 🙂

**Review by ahopkins — APPROVED** [review]

**Pull request merged**